plugins/org.eclipse.sirius/src/org/eclipse/sirius/business/internal/image/ImageManagerForWorkspaceResource.java

```diff
@@ -111,6 +113,27 @@ public CreateImageFileProvider getCreateFileFunc(EObject contextObject, String s
         return new CreateImageFileProvider(createFileFunction, filename);
     }
 
+    @Override
+    public String getImagesFolderName(EObject contextObject) {
+        String imageFolderName;
+        String platformString = contextObject.eResource().getURI().toPlatformString(true);
+        IFile airdFile = ResourcesPlugin.getWorkspace().getRoot().getFile(new Path(platformString));
+        IProject project = airdFile.getProject();
+        Optional<IFolder> optImageFolder = Optional.empty();
+        try {
+            optImageFolder = Arrays.stream(project.members()).filter(IFolder.class::isInstance).map(IFolder.class::cast)
```

Comment:
please try to indent the methode called on stream
Arrays.stream(project.members()).filter(IFolder.class::isInstance).map(IFolder.class::cast)
optImageFolder = Arrays.stream(project.members())//
            .filter(IFolder.class::isInstance)//
            .map(IFolder.class::cast)//

Reply:
Done